utils/dataHandling.ts

```diff
@@ -0,0 +1,277 @@
+import { load } from '@welshman/net';
+import { deriveProfile } from '@welshman/app';
+import { Router } from '@welshman/router';
+import { publishThunk } from '@welshman/app';
+import { makeEvent } from '@welshman/util';
+import { pubkey } from '@welshman/app';
+
+/**
+ * Establishes initial relay connections on app launch
+ * @returns Promise that resolves when connections are established
+ */
+export const establishRelayConnections = async (): Promise<void> => {
```

Comment:
This shouldn't really be necessary, connections are pretty quick, and should be managed mostly automatically. If a connection fails, welshman will automatically fall back on subsequent requests.